.changelog/changelog.tmpl

```diff
@@ -0,0 +1,56 @@
+{{- if index .NotesByType "breaking-change" -}}
+BREAKING CHANGES:
```

Comment:
Is this the correct order? In the past we've always placed Features at the top, right?

Reply:
I reprioritized features to be on top.  Other HashiCorp products have backward incompatibility changes above - but their features seem to be less significant than how we frame them in Nomad and seem more like improvements to me, e.g. https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#1100-beta3-may-27-2021. Updated it to reflect Nomad practices and we can change as we want.